templates/70400/Property.php.twig

```diff
@@ -0,0 +1,5 @@
+{{ prototype.visibility }}
+{%- if prototype.type.notNone %}
+ {{ prototype.type }} 
+{%- endif %}
+ ${{ prototype.name}}{% if prototype.defaultValue.notNone %} = {{ prototype.defaultValue.export }}{% endif %};
```

Comment:
I guess we can discuss further in the other PR: but would prefer if we used the standard version notation, e.g. `7.1`. We can then use `version_compare`, and `phpversion()` in the iterator.

Reply:
Additionally I wondered if we should namespace this, so `templates/php/<current templates>` `templates/php/7.1/<php 7.1 overrides>` etc. But as this space isn't used for anything else I guess it's fine.

We could also make the selection logic part of this library...